### ISSUE TITLE
How to run just Sails migrations, not server?

### ISSUE THREAD

**Issue by dbkaplun** [balderdashy/sails#3133]
Is it possible to run Sails DB migrations without the server? I'd prefer if I could do it via commandline but JS could work too.

**Comment by ryanwilliamquinn** [comment]
Maybe this package will help: https://github.com/building5/sails-db-migrate

**Comment by dbkaplun** [comment]
That's not exactly what I'm looking for. I saw that one and [sails-migrations](https://github.com/BlueHotDog/sails-migrations) (which doesn't work on v0.11) but the functionality I want is baked into Sails already. I want to run Sails' built-in `migrate: 'alter'` behavior in the shell or as JS.

**Comment by crh3675** [comment]
Just run sails.lift (with some other stuff)

**Comment by dbkaplun** [comment]
Can we make this a feature please? Something like `sails migrate` that just migrates and quits?

**Comment by sailsbot** [comment]
Thanks for posting, @dbkaplun.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
